client/app/scripts/superdesk/upload/crop-directive.js

```diff
@@ -36,6 +38,9 @@ define([
 
                 scope.$watch('src', function(src) {
                     elem.empty();
+                    if (scope.maxFileSize && ((scope.file.size / 1048576) > parseInt(scope.maxFileSize, 10))) {
```

Comment:
I think it can use some comment - maybe having a constant for the number and put there that `maxFileSize` is in MB